example/example.py

```diff
@@ -1,39 +1,90 @@
+'''
+This is a module help.  Put it into genarated C code.
+'''
+
 __author__ = 'Module Author'
 
 
+# These UPPERCASE variables are treated as constants in C code
+CONST1 = 1 + 10 
+CONST2 = 2.2 * 2.2
+CONST3 = "333" + '444'
+ONE_SHOT = False
+PERIODIC = True
+
+
+# We need types to generate proper variable declarations and function result in C code
 def func_a():
     pass
 
-def func_b(arg1):
+def func_b(arg1: int) -> None:
     pass
 
-def func_c(arg1, arg2) -> None:
-    pass
+def func_c(arg1: int, arg2: int) -> int:
+    res = arg1 + arg2 * arg2 
+    return res
```

Comment:
Do we want to keep this here? Other functions use `pass` to indicate this code does not get magically rewritten into C